### PR TITLE
Run local production server

### PR DESCRIPTION
**Pull request by Gaya**
Allows the user to run a local version of the production server.

## Description

Run a local version of production which communicates through the API provided by Docker.

## Motivation and Context

In order to test production builds locally we have to spin up a local server which can serve assets to the browser. This was not possible before, but with these changes you can! All you need is the Docker running in dev mode and it will be available to use in this local production build.

## How Has This Been Tested?

By running the local server commands.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (non-breaking change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by dennisoelkers — CHANGES_REQUESTED** [review]
Looks good so far, great work! Just one minor request.

File: graylog2-web-interface/runProductionLocal.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+const buildDir = process.argv[2] || 'target/web/build';
+const plugins = process.argv.slice(3);
+
+const pluginModules = plugins.map((plugin) => JSON.parse(fs.readFileSync(plugin)));
+const assets = productionHelpers.collectAssets(pluginModules);
+const pluginMounts = productionHelpers.collectMounts(plugins);
+
+const { url } = productionHelpers.bootstrapExpress(
+  buildDir,
+  productionHelpers.config('http://localhost:9000/api'),
```
[the remark this answers is not in the document]
Comment (Gaya) [reply graylog2-web-interface/runProductionLocal.js:17]: Yeah, good idea, will change

**Comment by Gaya** [comment]
@dennisoelkers did you get the chance to look at this one? I want to use it for testing in `cloud` too

**Comment by dennisoelkers** [comment]
Not requested again, closing it for now.

**Pull request closed without merging**